website/he.js

```diff
@@ -0,0 +1,170 @@
+jquery_lang_js.prototype.lang.nl = {
+	
+	// Header
+	
+	'Home':
+		'בית',
+	'Press':
+		'לחץ',
+	'FAQ':
+		'FAQ',
+	'Changelog':
+		'רשימת שינויים',
+	'Contact':
+		'צור קשר',
+	'Links':
+		'קישורים',
+	'Bug/feature tracker':
+		'Bug/feature tracker',
+	'Blog':
+		'בלוג',
+	'Language':
+		'שפה',
+	
+	// Footer
+	
+	'Copyright © 2015 Reason Company Software Inc.':
+		'Copyright © 2015 Reason Company Software Inc. Nederlandse vertaling: <a href="mailto:contact@studio384.be">Studio 384</a>, S. Paternotte.',
```

Comment:
what is this:

 Nederlandse vertaling: <a href="mailto:contact@studio384.be">Studio 384</a>, S. Paternotte.'
 ?